.github/workflows/main.yml

```diff
@@ -14,6 +14,10 @@ name: CI
 on:
   # Triggers the workflow on pull request events
   pull_request:
+    # The types trigger are needed to run the jobs again which are skipped for
+    # draft pull requests.
+    # https://github.community/t/dont-run-actions-on-draft-pull-requests/16817/16
+    types: [opened, synchronize, reopened, ready_for_review]
```

Comment:
So either we only use this and remove `if: ${{ github.event.pull_request.draft == false }}` or the other way around.

For right now I would keep `if: ${{ github.event.pull_request.draft == false }}` and if we decide to change this then do it in a separate PR. 

Anyways we do have actions right now that always run and for which it makes sense for me like `check-files-licence-headers`.

Reply:
My comment is a bit misleading. We need both. `if: ${{ github.event.pull_request.draft == false }}` and the types. The reason for this is:

If we have a draft pull request, all jobs with `if: ${{ github.event.pull_request.draft == false }}` will be skipped. When you switch from draft pull request to normal pull request, the jobs will **not** re-trigger (until you push something again). Therefore, it would be possible to have a draft pull request where the jobs will be skipped, switch to normal pull request and then merge into `main` without running our CI.

`types: [opened, synchronize, reopened, ready_for_review]` will re-trigger the jobs when you switch from draft pull request to normal pull request.

I will adjust the comment later 👍

Reply:
Done ✅